### PR TITLE
Remove jcenter from list of repositories

### PR DESCRIPTION
**Pull request by gastaldi**
See https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/ for more details

### CODE REVIEW

**Comment by slonopotamus** [comment]
Note that this is not the only dependency on jcenter/bintray in testcontainers.

As far as I understand, testcontainers currently publishes its releases via bintray: https://github.com/testcontainers/testcontainers-java/blob/master/gradle/bintray.gradle

**Comment by bsideup** [comment]
@gastaldi could you please remove it from everywhere, not only the top level file? There are a few entries (e.g. examples)

**Comment by gastaldi** [comment]
@bsideup My Gradle-fu is a bit limited, but I think I removed every reference from the repository, except for the `RELEASING.md`, which says that Bintray syncs with Central and I have no idea how to do that in Gradle :)

File: build.gradle [hunk 1]
```diff
@@ -2,7 +2,6 @@ plugins {
     id 'io.franzbecker.gradle-lombok' version '3.1.0'
     id 'nebula.provided-base' version '3.0.3'
     id 'com.github.johnrengelman.shadow' version '5.2.0'
-    id "com.jfrog.bintray" version "1.8.4" apply false
```
Comment (rnorth) [inline build.gradle:5]: It's not quite as easy as this; we need to replace our bintray publication with something else, and I'm afraid that this is a task that core maintainers need to do.

Removal of `jcenter()` from `repositories` blocks is still valuable, though!

**Review by rnorth — CHANGES_REQUESTED** [review]

**Comment by bsideup** [comment]
Seems to be superseeded by #4931, sorry! Thanks for your time working on this PR tho, much appreciated!

**Pull request closed without merging**